__tests__/articles.test.js

```diff
@@ -90,3 +90,114 @@ describe("GET/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET/api/articles/:article_id/comments", () => {
+  test("responds with a status of 200, returns an array of comments with the corrosponding article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
for `article_id` we can `expect(1)` because you're accessing article_id `1` for this test